mep/books/migrations/0024_add_work_sort_title.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 2.2.11 on 2020-04-22 17:54
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('books', '0023_merge_20200406_1206'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='creator',
+            options={'ordering': ['creator_type__order', 'order', 'person__sort_name']},
+        ),
+        migrations.AlterModelOptions(
+            name='creatortype',
+            options={'ordering': ['order']},
+        ),
```

Comment:
I thought that these changes were captured as part of earlier migrations, but apparently they weren't?

Reply:
We definitely added the order options, but I think we didn't create migrations to reflect the changes. (IMO it shouldn't be required since the db structure hasn't changed at all, but I know Django complains about it.)